kombu/transport/redis.py

```diff
@@ -38,6 +38,7 @@
 * ``unacked_restore_limit``
 * ``fanout_prefix``
 * ``fanout_patterns``
+* ``global_keyprefix``
```

Comment:
Could you please add description here and type of the options?

Reply:
@matusvalo Sure I can and I [would](https://github.com/celery/kombu/pull/1349#discussion_r657200096). I have some questions though:

* What kind of documentation we mean here? If you mean to have a description of what the `global_keyprefix` means, it is described [here](https://github.com/celery/kombu/blob/6d2ea9a0e7139b77100607c7c07eb246b39a8036/kombu/transport/redis.py#L489-L492) that is used by sphinx as I can see. If you meant another documentation, please advise where I can find that as I'm not a regular contributor at kombu. I just generated the documentation locally and the changes are reflected.
* What options do you refer to? Since it is a string variable, the value could be anything that the user sets.

Reply:
> What kind of documentation we mean here? If you mean to have a description of what the global_keyprefix means, it is described here that is used by sphinx as I can see.

The users of this transport are mainly interested in Transport Option they should be documented first.

> What options do you refer to?

Just add type and type and short description of ``global_keyprefix`` transport option
```suggestion
* ``global_keyprefix``: (str) The global key prefix to be prepended to all keys used by Kombu
```

Reply:
Thank you for the suggestion @matusvalo; now I see the way you wanted to write the docs.